.github/workflows/main.yml

```diff
@@ -0,0 +1,14 @@
+name: CI
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  build-package:
+    name: "Test Plugin"
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Test Plugin
+        run: swift package --allow-writing-to-package-directory format --lint
```

Comment:
Added a CI step to test that the plugin build and runs correctly -- as a bonus it also lints the plugin source files 😆

Reply:
This is set up correctly as far as I can tell, but it isn't running on this PR. Not sure why -- maybe it needs to be merged first and then it will run on subsequent PRs?

Reply:
> This is set up correctly as far as I can tell, but it isn't running on this PR. Not sure why -- maybe it needs to be merged first and then it will run on subsequent PRs?

See above comment.

Reply:
> Added a CI step to test that the plugin build and runs correctly -- as a bonus it also lints the plugin source files, and validates that `airbnb.swiftformat` and `swiftlint.yml` are formatted correctly (or the tool will fail to run).

Can we check if there are changes after this code runs so that we fail CI if the plugin source files don't conform to our style guide?

Reply:
Ideally the tool will emit an error exit code if `--lint` detects any issues, which would cause the CI job to fail. Will double-check that this actually works, once I get CI to run.

Reply:
With the changes in https://github.com/airbnb/swift/pull/168/commits/4408b6c4ffb904798477d26aad4a9b6efdc301f8 the tool / plugin emit a non-zero exit code when encountering a lint failure

Reply:
Tested this in #170, which added some intentional lint failures. The checks ran and then failed as expected: https://github.com/airbnb/swift/runs/7421227671?check_suite_focus=true